WordPress/gutenberg
# InputControl: Make `onChange` observable in Storybook

Required for testing IME keydown handling (follow-up to #59081)

## What?

Enhances the Storybook stories for InputControl so `onChange` events can be tested using the Actions panel.

## Why?

We generally want all our `onChange` events to be testable in Storybook.

## Testing Instructions

In [trunk Storybook](https://wordpress.github.io/gutenberg/?path=/story/components-experimental-inputcontrol--default), we can see that change events aren't logged in the Actions panel.

1. In [local Storybook](http://localhost:50240/?path=/story/components-experimental-inputcontrol--default) for this PR, the change events should now be logged as expected.
2. Enable the control for the `isPressEnterToChange` prop and reload.
3. The `onChange` should only be logged after blurring the input or hitting the Enter key. The `onValidate` calls should also be logged.
